website/docs/r/frontdoor.html.markdown

```diff
@@ -212,7 +211,11 @@ The `forwarding_configuration` block supports the following:
 
 * `cache_use_dynamic_compression` - (Optional) Whether to use dynamic compression when caching. Valid options are `true` or `false`. Defaults to `false`.
 
-* `cache_query_parameter_strip_directive` - (Optional) Defines cache behaviour in relation to query string parameters. Valid options are `StripAll` or `StripNone`. Defaults to `StripAll`.
+* `cache_query_parameter_strip_directive` - (Optional) Defines cache behaviour in relation to query string parameters. Valid options are `StripAll`, `StripAllExcept`, `StripOnly` or `StripNone`. Defaults to `StripAll`.
+
+* `cache_query_parameters` - (Optional) Specify query parameters (comma separated). Works only in combination with `cache_query_parameter_strip_directive` set to `StripAllExcept` or `StripOnly`.
```

Comment:
should this be an array of values rather then a string?

Reply:
@katbyte thank you, that's a good idea. In Front Door's `models.go` has `QueryParameters` the type `*string` that's why i've started as a string. But i agree that it's a much better to use an array here. I've made some changes to make `query_parameters` an array now.

![image](https://user-images.githubusercontent.com/17407022/129174016-ecf2d126-0a8b-4701-a674-e185b6bf38b7.png)